android/src/main/java/com/mapbox/reactnativemapboxgl/ReactNativeMapboxGLManager.java

```diff
@@ -59,6 +88,22 @@ public void setAccessToken(MapView view, @Nullable String value) {
         }
     }
 
+    @ReactProp(name = PROP_SET_TILT)
+    public void setTilt(MapView view, @Nullable double pitch) {
+        mapView.setTilt(pitch, 1L);
+    }
+
+    public static Drawable drawableFromUrl(String url) throws IOException {
+        Bitmap x;
+
+        HttpURLConnection connection = (HttpURLConnection) new URL(url).openConnection();
+        connection.connect();
+        InputStream input = connection.getInputStream();
+
+        x = BitmapFactory.decodeStream(input);
+        return new BitmapDrawable(x);
+    }
+
     @ReactProp(name = PROP_ANNOTATIONS)
```

Comment:
We should remove all annotations when this prop is called. If not, anytime the state of annotations changes, the previous state markers stay visible.